bot/cogs/snakes.py

```diff
@@ -18,29 +17,35 @@ def __init__(self, bot: AutoShardedBot):
     async def get_snek(self, name: str = None) -> Dict[str, Any]:
         """
         Go online and fetch information about a snake
-
         The information includes the name of the snake, a picture of the snake, and various other pieces of info.
         What information you get for the snake is up to you. Be creative!
-
         If "python" is given as the snake name, you should return information about the programming language, but with
         all the information you'd provide for a real snake. Try to have some fun with this!
-
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         :return: A dict containing information on a snake
+        
         """
+        
 
     @command()
-    async def get(self, ctx: Context, name: str = None):
+    async def get(self, ctx: Context, *, query: str = None):
         """
         Go online and fetch information about a snake
-
         This should make use of your `get_snek` method, using it to get information about a snake. This information
         should be sent back to Discord in an embed.
-
         :param ctx: Context object passed from discord.py
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         """
 
+        em = discord.Embed(title=str(query))
+        em.set_footer(text='Powered by wikipedia.org')
+        async with self.bot.http_session.get(f"https://en.wikipedia.org/w/api.php?format=json&action=query&prop=extracts&exintro=&explaintext=&titles={query}") as resp:
```

Comment:
You might want to have this string format defined on a separate line. Travis agrees.